ports/espressif/Makefile

```diff
@@ -469,11 +486,15 @@ update-all-sdkconfigs: $(BUILD)/esp-idf/config/sdkconfig.h
 update-board-sdkconfig: $(BUILD)/esp-idf/config/sdkconfig.h
 	python tools/update_sdkconfig.py --board=$(BOARD) --debug=$(DEBUG)
 
-BINARY_WIFI_BLOBS = libcore.a libespnow.a libmesh.a libnet80211.a libpp.a libsmartconfig.a libwapi.a
```

Comment:
Does `libmesh.a` have anything to do with ESP-NOW support? What is `libwapi.a` for ?
(I just don't know. I tried a websearch but didn't find an answer.)

Reply:
I don't actually know either. I just know the H2 doesn't have them available.